static/sass/custom/_linkable-heading.scss

```diff
@@ -0,0 +1,27 @@
+.linkable-heading {
```

Comment:
I am not convinced we need a wrapping element here. Could it think link be in the heading and then positioned to the left. This would reduced the complexity of his pattern.

Reply:
Can you use the same system as Vanilla. For example: `p-heading-archor` and imported via a mixin.

Reply:
Removing the wrapping element will probably work just as well but it'll be the third refactor plus a port from the whole codebase to this one - is it worth refactoring if it works as is now just to remove a wrapping div?